### PR TITLE
More star rating fixes

### PR DESCRIPTION
**Pull request by yunosh**
This is a follow-up to PR #21 fixing more issues with rating stars.

### CODE REVIEW

File: modules/tv/search.php [hunk 1]
```diff
@@ -281,7 +281,7 @@
             }
         // If the next thing starts with stars, it's a movie rating query
             if (preg_match('#(\\*+\s*(1/2\b|0?\.5\b|-)?)\s*(.*?)$#', $search_str, $match)) {
-                $starcount = substr_count($match[1], '*') / 4;
+                $starcount = substr_count($match[1], '*') / 5;
```
Comment (dekarl) [inline modules/tv/search.php]: Is this a magic value for max_stars? Defaults to 4, you suggest changing it to 5?

File: modules/tv/search.php [hunk 2]
```diff
@@ -281,7 +281,7 @@
...
                 if (preg_match('/1\\/2|\\.5|-/', $match[1]))
                     $starcount += 0.125;
```
Comment (dekarl) [inline modules/tv/search.php]: This appears to be a magic value for 1/(max_stars*2)

File: modules/tv/search.php [hunk 3]
```diff
@@ -571,19 +571,12 @@ function category_list() {
...
-            $frac = ($x * max_stars) - intVal($x * max_stars);
-            if ($frac >= .75)
-                echo '&frac34;';
-            elseif ($frac >= .5)
-                echo '&frac12;';
-            elseif ($frac >= .25)
-                echo '&frac14;';
-            echo '</option>';
+            echo '>', str_repeat(star_character, $x), '</option>';
         }
```
Comment (dekarl) [inline modules/tv/search.php]: Can you use max_stars*2 options and allow for half stars? Your suggested default of 5 stars would then nicely match to the common 1-10 star_rating scheme. (used in mythfrontend, imdb, tmdb, etc)

File: modules/tv/search.php (hunk 1 above)
Comment (yunosh) [inline modules/tv/search.php]: It's counting the number of asterixes entered in the search form. This patch just normalizes the maximum stars number to 5 stars, which is used anywhere else in mythweb.

File: modules/tv/search.php (hunk 2 above)
Comment (yunosh) [inline modules/tv/search.php]: See above

File: modules/tv/search.php (hunk 3 above)
Comment (yunosh) [inline modules/tv/search.php]: I actually plan to submit another patch to bump the maximum number of start to 10 all over mythweb, to match mythfrontend. For this PR I like to only fix the issues with the *current* number of stars in mythweb.

**Comment by yunosh** [comment]
Any update on this, any more changes necessary for this PR to be accepted?

**Comment by yunosh** [comment]
Sorry to be nagging, but another month has passed and without this fix, star search in mythweb is currently broken.
I wouldn't mind to get commit permissions to the mythweb repo and apply the fixes myself either.
